libnaboConfig.cmake.in

```diff
@@ -6,13 +6,7 @@
 # Compute paths
 get_filename_component(libnabo_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 set(libnabo_INCLUDE_DIRS "@libnabo_include_dirs@")
- 
-# Our library dependencies (contains definitions for IMPORTED targets)
```

Comment:
This should be tested with libpointmatcher. The reason why this was put in place is because otherwise libpoint-matcher (non catkin) compilation was not working. If it is now working I am fine with merging.